integr8ly/integreatly-operator
# MGDAPI-3659 - add new CRO release v0.38.0

# Issue link
[MGDAPI-3659](https://issues.redhat.com/browse/MGDAPI-3659)

# What
New CRO release v0.38.0

# Verification steps
- Install RHOAM on a CCS cluster.
- Confirm that a new version of CRO is present
- Confirm that Postgres, Redis and blobstorage resources are provisioned correctly.
- Verify upgrade is successful
